index.html

```diff
@@ -5,21 +5,30 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="styles.css">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link
+    rel="stylesheet"
+    href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+  />
     <title>Portfolio</title>
   </head>
   <body>
       <!--work section-->  
-      <header class="navbar">
-        <ul>
-          <li class="navbar-logo">Tadesse J.</li>
-          <li class="navbar-toggle"><i class="fa fa-bars"></i></li>
-          <li class="navbar-links">Portfolio</li>
-          <li class="navbar-links">About</li>
-          <li class="navbar-links">Work</li>
-          <li class="nav-icon" ><i class="fa fa-envelope-o"></i></li>
-        </ul>
-       </header>
+      <header>
+        <nav class="navbar">
+          <a href="#"><h2 class="nav-logo">Tadesse</h2></a>
+          <ul class="nav-menu">
```

Comment:
- [ ] I noticed that your mobile menu does not fit on the screen on the mobile view when I open the menu. I have to scroll horizontally to see the close icon. Please fix it 👍 .

![image](https://user-images.githubusercontent.com/63473584/179812675-61058602-38a1-49a9-83a2-d977148ef45b.png)

![image](https://user-images.githubusercontent.com/63473584/179812783-cb876a4f-739a-4bfc-a0b9-10be749a33d8.png)